### PR TITLE
Added Repeat configuration flag to support single invocation.

### PR DESCRIPTION
**Pull request by egieseke**
## Summary

Add configuration flag to loadgenerator utility to specify if it it should repeat after the first round.

## Test Plan

Tested manually.

### CODE REVIEW

File: cmd/loadgenerator/main.go [hunk 1]
```diff
@@ -177,12 +181,12 @@ func generateTransactions(restClient client.RestClient, cfg config, privateKey *
...
-				_, err = restClient.SendRawTransaction(txns[x])
-				if err != nil {
-					if strings.Contains(err.Error(), "txn dead") || strings.Contains(err.Error(), "below threshold") {
+				_, err2 := restClient.SendRawTransaction(txns[x])
+				if err2 != nil {
+					if strings.Contains(err2.Error(), "txn dead") || strings.Contains(err2.Error(), "below threshold") {
 						break
 					}
-					fmt.Fprintf(os.Stderr, "unable to send transaction : %v\n", err)
+					fmt.Fprintf(os.Stderr, "unable to send transaction : %v\n", err2)
```
Comment (tsachiherman) [inline cmd/loadgenerator/main.go:189]: the function doesn't return the `err` variable, so I can't see why you would need to allocate another `err2` variable.. anything that I'm missing ?

**Review by tsachiherman — CHANGES_REQUESTED** [review]
I think that adding a "-once" command line is more suitable than adding a configuration option.

File: cmd/loadgenerator/main.go (hunk 1 above)
Comment (egieseke) [reply cmd/loadgenerator/main.go:189]: I was seeing a memory segmentation fault when this is handling errors.

Comment (tsachiherman) [reply cmd/loadgenerator/main.go:196]: huh.. I think that you're correct in that it should be a unique variable per go-routine.
Let me confirm that tomorrow morning with race enabled.

File: cmd/loadgenerator/main.go [hunk 2]
```diff
@@ -36,6 +37,12 @@ import (
 
 const transactionBlockSize = 800
 
+var runOnce = flag.Bool("once", false, "Terminate after first spend loop")
+
+func init() {
+	flag.Parse()
```
Comment (tsachiherman) [inline cmd/loadgenerator/main.go:43]: I would have put this in the first line of `main`, but your implementation is not wrong.

**Comment by tsachiherman** [comment]
I will approve this after testing it with the -race flag. I'd hope to see that the issue you found is the only issue.

**Review by tsachiherman — APPROVED** [review]

**Pull request merged**